frontend/src/pages/RoomPage.tsx

```diff
@@ -257,27 +300,110 @@ export const RoomPage = () => {
         />
       </Stack>
       <Divider />
-      <Stack
-        direction="row"
-        spacing={2}
-        sx={{ width: "100%", flex: 1, minHeight: 0, p: 3 }}
-      >
-        <Stack spacing={2} sx={{ minWidth: "40%", maxWidth: "40%" }}>
-          <Box sx={{ flex: 1, minHeight: 0 }}>
-            <Question />
-          </Box>
-          <ChatProvider roomId={roomId || ""}>
-            <Box sx={{ height: "40%" }}>
-              <Chat />
+      <TabContext value={formType}>
+        <TabList
+          centered
+          onChange={handleChange}
+          sx={{
+            "& .MuiTabs-indicator": {
+              height: "0px",
+            },
+          }}
+        >
+          <Tab
+            label="Description"
+            value="description"
+            sx={{
+              fontWeight: "bold",
+              fontSize: "1.2rem",
+              textTransform: "none",
+            }}
+            icon={<Wysiwyg />}
+            iconPosition="start"
+          />
+          <Tab
+            label="Submission"
+            value="submission"
+            sx={{
+              fontWeight: "bold",
+              fontSize: "1.2rem",
+              textTransform: "none",
+            }}
+            icon={<DriveFolderUpload />}
+            iconPosition="start"
+          />
+        </TabList>
+        <TabPanel sx={{ p: 0 }} value="description">
+          <Stack
+            direction="row"
+            spacing={2}
+            sx={{ width: "100%", flex: 1, minHeight: 0, p: 3 }}
+          >
+            <Stack spacing={2} sx={{ minWidth: "40%", maxWidth: "40%" }}>
+              <Box sx={{ flex: 1, minHeight: 0 }}>
+                <Question />
+              </Box>
+              <ChatProvider roomId={roomId || ""}>
+                <Box sx={{ height: "40%" }}>
+                  <Chat />
+                </Box>
+              </ChatProvider>
+            </Stack>
+            <Box sx={{ flex: 1, minWidth: 0 }}>
+              <EditorProvider roomId={roomId || ""}>
+                <Editor language={"javascript"} />
+              </EditorProvider>
             </Box>
-          </ChatProvider>
-        </Stack>
-        <Box sx={{ height: "100%", flex: 1, minWidth: 0 }}>
-          <EditorProvider roomId={roomId || ""}>
-            <Editor language={"javascript"} />
-          </EditorProvider>
-        </Box>
-      </Stack>
+          </Stack>
+          <Stack
+            direction="row"
+            justifyContent="flex-end"
+            sx={{ py: 2, px: 3 }}
+          >
+            <StyledButton
+              label={"Submit Code"}
```

Comment:
Was thinking if we should put the "Submit Code" button directly below the "Leave Room" button instead. However, it might not be so easy to do so because the button won't be in the Tab Panel. We will need to define a conditional in the same stack where the TabContext is in to block the "Submit Code" button render on the Submission page.

Reply:
It's probably best to place the submit code button at the bottom right hand of the screen, so that its more familiar for people who have used LeetCode before.